pkgs/development/compilers/openjdk/jre.nix

```
@@ -7,7 +7,7 @@
 
 let
   jre = stdenv.mkDerivation {
-    name = "${jdk.name}-minimal-jre";
+    pname = "jre-minimal";
```

I think there are other jdk's.
```suggestion
    pname = "${jdk.pname}-minimal";
```